test/oauth_helper.js

```diff
@@ -9,45 +9,40 @@ const hapi = require('hapi')
 const url = require('url')
 const P = require('../lib/promise')
 
-module.exports = () => {
-  return new P((resolve, reject) => {
-    const api = new hapi.Server()
-    api.connection({
-      host: url.parse(config.oauth.url).hostname,
-      port: parseInt(url.parse(config.oauth.url).port)
-    })
+module.exports = async () => {
+
+    return new P((resolve, reject) => {
```

Comment:
this file also needs a 2 space indent back, seems like it was changed to 4 spaces